sass/helpers/spacing/margin-padding.sass

```diff
@@ -26,4 +26,11 @@ $current: $range-start
     margin-#{$side}: 0 !important
   
   .is-#{$side}-paddingless
-    padding-#{$side}: 0 !important
\ No newline at end of file
+    padding-#{$side}: 0 !important
+
+@each $side in (left, right)
+  has-margin-#{$side}-auto
```

Comment:
nth: Add class for margin auto for all sides:
`.has-margin-auto
  margin: auto !important
`

Reply:
Added this class as well 😄